data/siteText.yaml

```diff
@@ -62,7 +61,7 @@ envEffects-p2: >-
   nonylphenol, and antimicrobials have been associated with cancer and endocrine disruption. 
   The high sorption capacity of plastic enables the accumulation of persistent organic pollutants 
   such as polychlorinated biphenyls, polycyclic aromatic hydrocarbons, and organochlorine pesticides, 
-  with concentrations of 10^5 – 10^6 times higher than in the surrounding water column. Trace metals 
+  with concentrations of 10<sup>5</sup> – 10<sup>6</sup> times higher than in the surrounding water column. Trace metals 
```

Comment:
ah, it is probably escaping the html bits, we should be doing `{{{ }}}` for all the text, but this one might be missing it.